ColorPair.h

```diff
@@ -0,0 +1,35 @@
+#ifndef COLORPAIR_H
+#define COLORPAIR_H
+
+#include <iostream>
+#include <string>
+
+namespace TelCoColorCoder {
+    enum class MajorColor { WHITE, RED, BLACK, YELLOW, VIOLET };
+    enum class MinorColor { BLUE, ORANGE, GREEN, BROWN, SLATE };
+
+    class ColorPair {
+    public:
+        ColorPair(MajorColor major, MinorColor minor);
+        MajorColor getMajor() const;
+        MinorColor getMinor() const;
+        std::string toString() const;
+
+    private:
+        MajorColor majorColor;
+        MinorColor minorColor;
+    };
+
+    class ColorUtils {

```

Comment:
Nice approach to keep functionality related to `ColorPair` in a single file. However, can you think of a more specific name for `ColorUtils`? Why should all utils (conversions, printing, etc) be present in a single file?
Motivation: Names like "utils" tends to collect all functionality, making it hard for multiple developers to work independently.

Reply:
I have changed the name of ColorUtils to ColorOperations.

Reply:
Yes, better than Utils :)